src/main/java/htsjdk/variant/variantcontext/GenotypeBuilder.java

```diff
@@ -185,14 +186,31 @@ public final void reset(final boolean keepSampleName) {
      * created, althrough the contents of array values like PL should never be modified
      * inline as they are not copied for efficiency reasons.
      *
+     * Note: if attributes are added, the new Genotype will be modified
```

Comment:
`if attributes are added` -> `if attributes are added via this builder after a call to make()`